src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -6,8 +6,16 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        // REVIEW NEEDED: AVX support cannot be checked in .NET Core App 2.0, so we assume Vector128 alignment for SSE.  Is it okay?
```

Comment:
This should probably say `netstandard2.0` and not `.NET Core App 2.0`.  This code is also used on the Windows-only .NET Framework.